### PR TITLE
Merge branch-PPP-Qiaoran with Master

### LINKED ISSUE #6: Create individual Project Portfolio Pages

### PR DESCRIPTION
**Pull request by Qiaoran-M**

### CODE REVIEW

File: docs/team/Qiaoran.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        * to be added soon
+
+* **Contributions to team-based tasks**:
+    * to be added soon
+
+* **Community**:
+    * Review/Mentoring contributions:
+        * to be added soon
+    * Contributions beyond the project team:
+        * to be added soon
```
Comment (lfrostbytee) [inline docs/team/Qiaoran.md]: As specified under the [failing CI checks](https://github.com/AY2223S1-CS2103-F09-2/tp/pull/19/checks), I think we will need to add an extra newline character to the end of every file (Hence, why this file failed the CI check as well).

**Review by lfrostbytee — COMMENTED** [review]
Other than the styling errors on the PPP, I think the changes made to UserGuide.md may have to be re-done on a separate branch and pull request similar to #11 which you can eventually link to #7.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -181,12 +181,12 @@ _Details coming soon ..._
...
+| **Clear**  | `clear`                                                                                                                                                               |
+| **Delete** | `delete INDEX`<br> e.g., `delete 3`                                                                                                                                   |
+| **Edit**   | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`                                           |
+| **Find**   | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`                                                                                                            |
+| **List**   | `list`                                                                                                                                                                |
+| **Help**   | `help`                                                                                                                                                                |
```
Comment (lfrostbytee) [inline docs/UserGuide.md]: Perhaps these changes can be done a separate branch and pull request

Comment (Benjamin-Sim) [reply docs/UserGuide.md]: I merged your `branch-UG-Qiaoran` with `Master` yesterday, see #12. Maybe you can try adding your new changes to `UserGuide.md` to that branch and see if #12 updates? I'm not sure if a merged PR will show new changes or if you need to create a new PR to see the new changes.

File: docs/team/Qiaoran.md (hunk 1 above)
Comment (Qiaoran-M) [reply docs/team/Qiaoran.md]: thank you! I will add the newline character to this file.

**Review by lfrostbytee — APPROVED** [review]
Lgtm!

**Comment by lfrostbytee** [comment]
Unfortunately, I think I didn't review your PPP properly as it does not seem to be showing (same issue as #24). Feel free to see my comments there.

**Pull request merged**